### PR TITLE
Feature gil add whoislib

### PR DESCRIPTION
**Pull request by dhkron**

### CODE REVIEW

File: whois/parsers/_converters.py [hunk 1]
```diff
@@ -61,10 +61,15 @@
     converter.Dummy(),
 ]
 
+email_converter = [
```
Comment (wavenator) [inline whois/parsers/_converters.py]: rewrite to email_converters

File: whois/parsers/_matchers.py [hunk 2]
```diff
@@ -477,11 +477,6 @@
             flags=re.IGNORECASE,
         ),
         matcher.Regex(
-            pattern=r'registrar_name:\s*(?P<value>.+)',
```
Comment (wavenator) [inline whois/parsers/_matchers.py]: why removed?

File: whois/parsers/resources/registrars.py [hunk 3]
```diff
@@ -8,6 +11,7 @@ class Registrars:
         https://www.icann.org/registrar-reports/accredited-list.html
     '''
     registrars = []
+    registrars_file_path = os.path.join(os.path.dirname(__file__), 'registrars_names')
```
Comment (wavenator) [inline whois/parsers/resources/registrars.py]: change to tempdir

Comment (dhkron) [inline whois/parsers/resources/registrars.py]: done

**Comment by dhkron** [comment]
I removed the "registrars_names" file from the repo

**Comment by wavenator** [comment]
Added abuse email subfield. Registrars file updates regularly

**Pull request merged**